deployment/cdk/opensearch-service-migration/lib/stack-composer.ts

```diff
@@ -457,7 +458,7 @@ export class StackComposer {
         }
 
         let trafficReplayerStack
-        if ((trafficReplayerServiceEnabled && networkStack && migrationStack) || (addOnMigrationDeployId && networkStack)) {
```

Comment:
If we don't have the MigrationAssistanceStack we don't know if there is a valid EFS instance for the shared logs.  I think this requirement makes the _addOn_ deployments no longer possible.  Not sure if this is a blocker or not.